bpf/lib/lb.h

```diff
@@ -1636,6 +1719,11 @@ static __always_inline int lb4_local(const void *map, struct __ctx_buff *ctx,
 	 * to make it appear from an outside address.
 	 */
 	if (saddr == backend->address) {
+	#if defined(ENABLE_LOCAL_REDIRECT_POLICY) && defined(HAVE_NETNS_COOKIE)
+		if (netns_cookie > 0 && unlikely(lb4_svc_is_localredirect(svc)) &&
+		    lb4_skip_xlate_from_ctx_to_svc(netns_cookie, tuple->daddr, tuple->sport))
+			return CTX_ACT_OK;
+	#endif /* ENABLE_LOCAL_REDIRECT_POLICY && HAVE_NETNS_COOKIE */
```

Comment:
This shouldn't be under `#ifndef DISABLE_LOOPBACK_LB`.

Reply:
Done. 

I can move the check for LRP loopback into a separate block if thats more readable. That would add an additional (saddr == backend->address) check.